templates/terraform/examples/cloud_identity_group_membership_user.tf.erb

```diff
@@ -14,10 +13,9 @@ resource "google_cloud_identity_group" "group" {
 }
 
 resource "google_cloud_identity_group_membership" "cloud_identity_group_membership_basic" {
-  provider = google-beta
   group    = google_cloud_identity_group.group.id
 
-  member_key {
+  preferred_member_key {
```

Comment:
Why is the name changing here?

Reply:
member_key is only supported in the beta API. This test is being run against the GA API; member_key usage is instead handled by handwritten tests in third_party/terraform/tests/resource_cloud_identity_group_membership_test.go.erb